among_us_stats.Rmd

```diff
@@ -0,0 +1,171 @@
+# Among Us Player Statistics
+
+Myles Ingram
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+```{r include=FALSE}
+library(ggplot2)
+library(dplyr)
+library(tidyverse)
+library(plyr)
+require(scales)
+```
+
+This R code runs player level and community level statistics on Among Us Player data (which can be found on the home page of the app). To use this script, add your name and a userid to the among_us_player_stats.csv along with your player stats. Then enter your username and userID into the code, run the script and take a look! This code will use my username as an example.
+```{r}
+plyr_stats <- read_csv("../resource/among_us_player_stats.csv")
+Username = "Big Cheese"
```

Comment:
Please double check this link, I think it should be 

resources/among_us_stats/among_us_player_stats.csv

Reply:
See above